provisioning/wildfly_channel_in_zips.adoc

```diff
@@ -0,0 +1,210 @@
+* https://github.com/wildfly-extras/prospero/issues/754[prospero#754] - Add ability to specify a "target" version for updates
+* https://issues.redhat.com/browse/WFMP-271[WFMP-271] Add a <name> parameter to the <channel> configuration
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-19130[WFLY-19130] - [Community] Publish Wildfly channel and manifest during project build
+** http://docs.wildfly.org/wildfly-proposals/build/WFLY-19130_publish_Wildfly_channel_manifest.html[Analysis Document]
+
+=== Stability Level
+
+* [x] Experimental
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+* mailto:bstansbe@redhat.com[Brian Stansberry]
+* mailto:bspyrkos@redhat.com[Bartosz Spyrko-Smietanko]
+
+=== Testing By
+
+* [x] Engineering
+
+=== Affected Projects or Components
+
+* https://github.com/wildfly/wildfly[WildFly]
+** WildFly distributions will incorporate channel metadata used to create the distributions and tooling to update the installation to a more recent version.
+* https://github.com/wildfly-extras/prospero[Prospero]
+** Prospero will be used as the library to update WildFly installations but will not surface to the user (the `installation-manager` script is the user entry point)
+* https://github.com/wildfly/wildfly-maven-plugin[wildfly-maven-plugin]
+** The WildFly Maven Plug-in provides the functionality to provision WildFly to create the distributions with the expected channel metadata.
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
+
+* [x] Traditional standalone server (unzipped)
+
+== Requirements
+
+* WildFly distributions archives (zip & tgz) must contain the channel metadata that describes their installation state.
+** tracked by https://issues.redhat.com/browse/WFLY-19221[WFLY-19221]
+** Provisioned feature packs remain referenced by their Galleon feature-pack-locations (e.g. `wildfly@maven(org.jboss.universe:community-universe):current#33.0.0.Final`).
+*** Users that use Galleon tool to udpate their WildFly installation will continue to be able to do so.
+*** Galleon universe artifacts (`org.jboss.universe:community-universe` & `org.jboss.universe.producer:wildfly-producers`) will be added to the WildFly channels
+* WildFly distributions archives (zip & tgz) must contain the tooling to let users update their installations
+** tracked in https://github.com/wildfly-extras/prospero/issues/749[prospero #749]
+** Users must be able to list and select the updates to apply to their installations
+** The user MUST specify the updates to apply.
+** As this feature is experimental, the tooling should warn the user that updating their installation is an experimental mechanism
+** these operations will be using Prospero that needs to be integrated as a JBoss module in the WildFly distributions.
+* Updates must not discard any user changes to an installation (in their configuration files or JBoss modules directory)
```

Comment:
Note - currently the files under `modules/system` are considers "system-path" and Galleon updates take precedence over user changes

Reply:
I think that's fine. We don't support users modifying system modules, so if they do they can't use this feature.